yt/yt/server/job_proxy/user_job_write_controller.cpp

```diff
@@ -126,6 +126,14 @@ class TMultiplexingWriter
 
             return
                 BIND([writer = Owner_->Underlying_, rowBuffer = RowBuffer_, rows = std::move(Rows_)] {
+                    // NB: In multiplexing writer, there are multiple value consumers pointing to the same writer.
```

Comment:
There is a similar code https://github.com/ytsaurus/ytsaurus/blame/49e9d43be87c9973667714c25722258e48d94af6/yt/yt/client/table_client/value_consumer.cpp#L335 for the regular value consumers